scripts/wasm-rebuild/docker-scripts/rebuild_current.sh

```diff
@@ -41,7 +41,7 @@ if [ -d jsoncpp ]; then
 fi
 
 set +e
-scripts/travis-emscripten/build_emscripten.sh
+scripts/*/build_emscripten.sh
```

Comment:
Why not `ci` explicitly?

Reply:
Because this should also work on older versions of the repo.